sdk/src/main/java/co/omise/android/models/SourceType.kt

```diff
@@ -36,6 +36,18 @@ sealed class SourceType(
         object Ktc : Installment("installment_ktc")
         object KBank : Installment("installment_kbank")
         data class Unknown(@JsonValue override val name: String?) : Installment(name)
+
+        companion object {
+            fun availableTerms(installment: Installment): List<Int> =
+                    when (installment) {
+                        Bay -> listOf(3, 4, 6, 9, 10)
```

Comment:
Not sure if we have any other options but this is gonna make our code very inflexible

Reply:
That right, this for the case that the user want to create their own config rather than using installment terms from Capability API. So, this will help to not make a mistake from set the unavailable installment terms. 🙂

Reply:
👌 👌 👌